### DISCUSSION TITLE
Best way to suppress logs?

### DISCUSSION THREAD

**Discussion by n-at-han-k** [glaucocustodio/tanakai#9] [Q&A]
I’m working on an Instagram scraper that takes user input through the terminal.
I’d like to turn all logs off. Is there a recommended to do this?

**Comment by glaucocustodio** [comment]
perhaps setting `config.logger = nil` in `config/application.rb`?

**Reply by n-at-han-k** [reply]
> perhaps setting `config.logger = nil` in `config/application.rb`?

This is a purely terminal application, no rails.
EDIT: just seen the project generator in the README. Didn't realise that was a feature.
